modules/scheduler-impl/src/main/java/org/opencastproject/scheduler/impl/SchedulerServiceImpl.java

```diff
@@ -1081,7 +1081,7 @@ public int getEventCount() throws SchedulerException {
   public List<MediaPackage> findConflictingEvents(String captureDeviceID, Date startDate, Date endDate)
       throws SchedulerException {
     try {
-      final Organization organization = securityService.getOrganization();
+      final Organization organization = new DefaultOrganization();
```

Comment:
What was the reason for this change? It's possible that this might be breaking conflict checks for multitenant systems (as this organization is later used to look for conflicting events in the database).